WTWSMS/common/objectives/00_focuses.txt

```diff
@@ -209,7 +214,7 @@ focus_seduction = {
 		}
 		
 		modifier = {
-			factor = 0.01 # Faithful spouse/consort
+			factor = 0 # Faithful spouse/consort
```

Comment:
It's unrealistic for a failthful AI to suddenly become unfaithful after decades of loyalty. Most people do not do that if they are truly faithful.